wye/workshops/forms.py

```diff
@@ -43,7 +43,7 @@ def get_organisations(self, user):
     def clean_expected_date(self):
         date = self.cleaned_data['expected_date']
         if not (date > datetime.date.today() + datetime.timedelta(days=14)):
-            raise ValidationError('Expected date has to be atleast 2 weeks ahead from now')
+            raise ValidationError('This has to be atleast 2 weeks ahead from now')
```

Comment:
Can we make it better
"Workshop request has to future date and atleast 2 weeks ahead of today"